src/shop-app.html

```diff
@@ -242,21 +234,18 @@
 
     <app-header role="navigation" id="header" effects="waterfall" condenses reveals>
       <app-toolbar>
-        <div class="left-bar-item">
-          <paper-icon-button
-              class="menu-btn"
-              icon="menu"
-              on-tap="_toggleDrawer"
-              aria-label="Categories">
-          </paper-icon-button>
-          <a class="back-btn" href="/list/[[categoryName]]" tabindex="-1">
-            <paper-icon-button icon="arrow-back" aria-label="Go back"></paper-icon-button>
+        <div class="icon-button-container">
+          <a href="#" class="shop-icon-button menu-btn" on-click="_toggleDrawer" aria-label="Categories">
```

Comment:
Normally `href="#"` would cause the page to scroll to the top.  `iron-location` seems to do something to remove the `#` if it has no trailing string.  So that's why you don't see the `#` in the url when you click on `<a href="#" class="shop-icon-button menu-btn"` in the shop app.  The removal of `#` that I am not sure if it's a bug in app-route.

`href` or `href=""` would reload the current url.  But `iron-location` has a _globalOnClick to prevent  all the `<a>` to reload the page so it can do client side routing, which is expected.

So if we are looking for no-op from `<a>` I think it makes sense to use `href`.